src/engine/client/index.ts

```diff
@@ -139,6 +139,13 @@ export const createClient = ({
           })
         },
       },
+      locales: {
```

Comment:
Could we also use the CDA to fetch locales?
https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/locales/locale-collection/get-all-locales-of-a-space/console/js

Reply:
That would mean that we need to ask for CDA token for the apply command, wouldn't it? I am not sure if we want that

Reply:
Good point @Vinz93